### ISSUE TITLE
Add deploy to heroku button for demos

### ISSUE THREAD

**Issue by jettify** [aio-libs/aiohttp-demos#41]
It is very easy to deploy aiohttp app to heroku, and best part it is completely free. I think we should deploy few demos so users can play with apps.

Process is strait forward  basically create `Procfile` with:
```
web: mlserve -c models.yml -H 0.0.0.0 -P $PORT
```
And deploy app with git:
```
$ heroku create
$ git push heroku master
$ heroku open
```

**Comment by Arfey** [comment]
something like this https://github.com/britter/spring-boot-heroku-demo?

**Comment by jettify** [comment]
Yeah, something like that. Helps to deploy project in no time basically.
